.github/workflows/ci.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Test with the dotnet CLI
         run: dotnet test tests 
         env:
-          TEST_GITHUB_TOKEN: ${{secrets.TEST_GITHUB_TOKEN}}
\ No newline at end of file
+          GH_TOKEN: ${{secrets.TEST_GITHUB_MG}}
```

Comment:
To be updated once IT creates the new token (could be in another PR)

Reply:
Yes once I have the token I will come update this here